Sdtrig.tex

```diff
@@ -319,6 +319,37 @@ \subsection{Cache Operations}
     \end{steps}
 \end{commentary}
 
+\subsection{Invalid Virtual Addresses}
+
+For address matches without a mask, \RcsrTdataTwo must be able to hold all valid
+addresses in all supported translation modes. That means that after writing any
+of these valid addresses, the exact same value is read back, including any high
```

Comment:
=>  "the exact same XLEN-wide value is read back."

Reply:
Done.